docs/source/installation.rst

```diff
@@ -51,3 +51,10 @@ cloned this repo, run the following:
 .. code-block:: bash
 
    python3 -m pip install -r requirements.txt
+
+.. warning::
+    If you want to contribute to ``manim-community`` and have cloned the
+    repository to your local device, please uninstall the pip-installed version
+    of ``manim-community``, if you had installed it previously.
+    This is to avoid any accidental usage of the pip-installed version when developing
+    and testing on your local copy of the repository.
```

Comment:
To prevent this from happening, I use virtual environments.
E.g. with Anaconda it is very easy to create multiple virtual environments for python, and switching between them.
So maybe the sentence:
"Alternatively, you can use a **different  virtual python environment** for the cloned   ``manim-community`` repository than for the pip-installed version

Reply:
We are gonna use poetry for this #165

Reply:
> "Alternatively, you can use a different virtual python environment for the cloned manim-community repository than for the pip-installed version

I'm not sure if this would work everywhere. IIRC, @philippimhof set it up the same way and still had the problem.

Mentioning them once more : @philippimhof

Reply:
Works like a charm now. I sabotaged myself, forgetting there was an alias in my `. bashrc`, so the virtual environment could not work.